portal-api/src/v1/controllers/mandatoryCriteria.controller.js

```diff
@@ -21,9 +21,10 @@ const findOneMandatoryCriteria = async (version, callback) => {
   if (typeof version !== 'string') {
     throw new Error('Invalid Version');
   }
+  const versionAsNumber = Number(version);
```

Comment:
Do we want to check that this isn't `nan` here?

We probably want to add the conditional check on line 21 to `typeof version !== 'string' || isNaN(version)`